### PR TITLE
PLU-261: [onboarding-demo-experiment-6]: add ui changes based on design feedback

### PR DESCRIPTION
**Pull request by m0nggh**
## Problem
New users are confused with how to get started on Plumber, so GGWP v1 is an onboarding demo flow we are experimenting with to provide more help for users to onboard with Plumber.

## Details
Frontend
- Center modals
- Change demo tooltip
- Adjust modal padding
- Adjust demo sidebar position

## Tests
- [x] Old pipes still work
- [x] Demo pipe flow work as intended

### CODE REVIEW

**Comment by m0nggh** [comment]
* **#622** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/622?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#621** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/621?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#620** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/620?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#619** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/619?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#618** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/618?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#617** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/617?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop-v2`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @m0nggh and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/src/components/FlowRow/DemoFlowModal.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default function DemoFlowModal(props: DemoFlowModalProps): ReactElement {
     ? DEMO_VIDEOS_MAP[demoVideoId]
     : DEMO_VIDEOS_MAP[FORMSG_POSTMAN_TEMPLATE]
 
-  const [showVideoModal, setShowVideoModal] = useState(false)
+  const [showVideoModal, setShowVideoModal] = useState(!isPreCreated ?? false)
```
Comment (pregnantboy) [inline packages/frontend/src/components/FlowRow/DemoFlowModal.tsx]: the ?? false is a no-op since !isPreCreated will never be nullish

**Review by pregnantboy — APPROVED** [review]
looks ok, pending final design review from stacey

File: packages/frontend/src/components/FlowRow/DemoFlowModal.tsx (hunk 1 above)
Comment (m0nggh) [reply packages/frontend/src/components/FlowRow/DemoFlowModal.tsx]: yes boss got it

**Pull request merged**